addons/udes_stock/models/product_template.py

```diff
@@ -14,3 +21,16 @@ class ProductTemplate(models.Model):
 
     # Default to being a stockable product
     type = fields.Selection(default="product")
+
+    u_speed_category_id = fields.Many2one(
+        comodel_name="product.category",
+        domain=_domain_speed_category,
+        string="Product Category Speed",
+        help="Product category speed to match with location speed.",
```

Comment:
This field is added in code but not the view, is it needed?

We could also do with making the help string a bit clearer.

Reply:
It is needed, the speed category is assigned in the item master record class. The view should have been included in `product_template.xml`, I just missed it. About rewording the help string - I am not quite sure what the speed refers to? Is it the speed with which the product can be transported across the warehouse?